jobs/involuntary-dissolutions/involuntary_dissolutions.py

```diff
@@ -271,6 +272,9 @@ def stage_2_process(app: Flask):
             batch_processing.notes = 'Moved back into good standing'
             app.logger.debug(f'Changed Batch Processing with id: {batch_processing.id} status to Withdrawn.')
         batch_processing.last_modified = datetime.utcnow()
+        if batch_processing.meta_data is None:
```

Comment:
Don't think it's the correct place to add `stage_2_date` since the value will be populated for both business moving to stage 2 and being withdrawn from dissolution. Btw, @argush3 do we need to populate date for withdrawn dissolution?

Reply:
good point.  yeah i don't think `stage_2_date` should be populated here as per Hongjing's note about the withdrawn scenario.

It would be nice to include withdrawn date but i think we can leave it out for now.  We can determine it via the last_modified date for now.  And also, it would require us to update the AR filer processor code if we wanted that.  Let's try to minimize the work for now.

Reply:
Updated

Reply:
Your update is still incorrect.  The withdrawn flow is when there shouldn't be a `stage_2_date` populated

Reply:
Updated